apps/video/js/video.js

```diff
@@ -44,6 +44,8 @@ dom.player.mozAudioChannelType = 'content';
 
 function $(id) { return document.getElementById(id); }
 
+init();
```

Comment:
nit: for readability, we should move init call after all variables are declared ( Line 114)